### ISSUE TITLE
Incorrect row height with rowSpan, dontBreakRows and keepWithHeaderRows

### ISSUE THREAD

**Issue by Kasheftin** [bpampuch/pdfmake#1600]
I have a table with multiple rowSpans.
Some rows have incorrect height - the row height is larger than required, it seems there's an empty line at the bottom. I can not find any pattern of this issue, some rows have correct height, the other ones don't.

That's the simplest code for this issue for pdfmake playground:
```
dd = {
  "content": [
    {
      "table": {
        "headerRows": 1,
        "dontBreakRows": true,
        "keepWithHeaderRows": 1,
        "body": [
          ["1", "2", "3", "4"],
          [{
              "rowSpan": 2,
              "text": null
            }, null, null, null],
          [null, null, null, null],
          [{
            "rowSpan": 2,
            "text": null
          }, null, null, null],
          [null, null, null, null],
       ]
      }
    }
  ]
}
```
That's the result:
![screenshot from 2019-01-07 14-35-28](https://user-images.githubusercontent.com/379267/50768616-b6e82f00-1289-11e9-9620-3a14464cf96e.png)

That's the valid result of the code above, on my sight:
![screenshot from 2019-01-07 14-38-08](https://user-images.githubusercontent.com/379267/50768720-1f371080-128a-11e9-8a68-abcc961467e2.png)

**Comment by cemg** [comment]
If you want equal heights, you must set the keepWithHeaderRows value to 0.
keepWithHeaderRows property keeps row data with header rows, when your data is longer than one page and header and row data are different pages.

**Comment by Kasheftin** [comment]
Yes, it works as expected with headerRows=0. But why headerRows causes the bug?
In my real app headerRows=1 rule is also required.
